.k8s/README.md

```diff
@@ -367,3 +367,279 @@ There are a few different ways to deal with the problem. Here are just two of th
 Both of these approaches allow us to avoid storing original secret values *directly* inside manifests. However there are cases when it is acceptable to store `Secret` manifests (along with the original secret values) directly inside Git repository. This is when manifests are used only for development purposes (some experiments for instance) and are designed for being deployed in a development cluster and not in production. However the main problem with such an approach is that it can lead to having two sources of truth (read two sets of manifests) - one for development and one for production. In our opinion such approach should be avoided as much as possible (instead we should use production ways to deal with secretes even for development). With all that being said, when such *insecure* is used all the secret values may be shared by developers and the secret values are not *really* hidden. To make the rest of the manifests as environment agnostic as possible we should use [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) for deploying them to comply with production configurations. Pods just reference these secrets not knowing what their sources are thus making the rest of the manifests the same both for both development and production.
 
 For more details see [Good practices for Kubernetes Secrets](https://kubernetes.io/docs/concepts/security/secrets-good-practices/) and the blog [How to use AWS Secrets & Configuration Provider with your Kubernetes Secrets Store CSI driver](https://aws.amazon.com/ru/blogs/security/how-to-use-aws-secrets-configuration-provider-with-kubernetes-secrets-store-csi-driver/).
+
+
+
+## 9. Installing PostgreSQL with metrics view via Grafana
+
+
+### Prerequisites
+
+Delete the namespace ```example-api```
+
+```
+kubectl delete namespace example-api
+```
+
+### Steps
+
+#### 9.1. Install PostgreSQL via Helm Chart with postgres_exporter tool
+
+**From CLI**
+
+```
+helm repo add bitnami https://charts.bitnami.com/bitnami
```

Comment:
please add links for all the mentioned tools (helm, bitnami Postgres, Prometheus, yq, postgres_exporter, Grafana)